ts/src/base/Precise.ts

```diff
@@ -281,6 +281,73 @@ class Precise {
         }
         return (new Precise (string1)).le (new Precise (string2));
     }
+
+    static stringArithFromNumbers (number1: number, number2: number, methodTail: string) {
+        const string1 = numberToString (number1);
+        const string2 = numberToString (number2);
+        const method = 'string' + methodTail;
+        return this[method] (string1, string2);
```

Comment:
@samgermain let's avoid dynamic calls here. Very error-prone, less performant and we have no reason at all for having them if we can easily do it "statically"